### PR TITLE
Structure and revhistory cleanup

### PR DESCRIPTION
**Pull request by tomschr**
### Description

I've added you to this PR as I've discovered some smaller inconsistencies in our assemblies. Most of the changes that I did are minor, but I would like to get your final approval in case I forgot something.

Smart Docs maintainers, could you please review _your file(s)_ and tell me if you have any concerns etc.? If you are fine with the changes, approve this PR. Thank you!

These are the overall changes of this PR:

* Add `xml:id` on `revhistory`, using an "rh-" prefix
  (it creates the respective file name for the revision history)
* Add a few missing `xml:id`'s on structure
  (needed as it also creates the file name)
* Consolidate `<output/>` and combine it with `<structure>` (just a few)
  (technically fine, but easier to parse when done consistently)
* Remove title
  (as recommended by stylewhich committee)
* Remove `<revnumber>`, in case it's there
  (mostly useless as we don't use revision numbers)

### Questions/TODOs:

* [x] These articles have revision dates in 2024:

      articles/SELinux-policy.asm.xml
      articles/SLE_Micro_admin.asm.xml
      articles/elemental-teal.asm.xml
      articles/freeradius-setup-server.asm.xml
      articles/manage-vm-on-commandline.asm.xml
      articles/pam.asm.xml
      articles/security-slemicro.asm.xml
      articles/ulp-livepatching.asm.xml

* [x] These articles still have the description in revhistory "Describe the purpose of this revision":

      articles/SELinux-policy.asm.xml
      articles/SLE_Micro_admin.asm.xml
      articles/elemental-teal.asm.xml
      articles/freeradius-setup-server.asm.xml
      articles/manage-vm-on-commandline.asm.xml
      articles/pam.asm.xml
      articles/security-slemicro.asm.xml
      articles/ulp-livepatching.asm.xml

* [x] These articles have a wrong date for `<meta name="updated"/>`:

      articles/SELinux-policy.asm.xml
      articles/SELinux.asm.xml
      articles/SLE_Micro_admin.asm.xml
      articles/elemental-teal.asm.xml
      articles/pam.asm.xml
      articles/raid.asm.xml
      articles/security-slemicro.asm.xml
      articles/supportconfig.asm.xml
      articles/systemd-basics.asm.xml
      articles/ulp-livepatching.asm.xml

### CODE REVIEW

File: articles/cockpit-alp.asm.xml [hunk 1]
```diff
@@ -145,12 +145,10 @@ type="application/relax-ng-compact-syntax"?>
...
     </resource>
   </resources>
-  <structure>
-    <output renderas="article" xml:id="cockpit"/>
+  <structure xml:id="cockpit" renderas="article">
     <merge>
       <title>Administering &productname; using &cockpit;</title>
-      <revhistory>
-        <title>Changelog</title>
+      <revhistory xml="rh-cockpit">
```
Comment (tbazant) [inline articles/cockpit-alp.asm.xml]:
```suggestion
      <revhistory xml="rh-alp-cockpit">
```

**Review by janajaeger — APPROVED** [review]

**Review by tbazant — APPROVED** [review]
one small suggestion, THANKS Toms for aligning the docs

**Review by jfaltenbacher — APPROVED** [review]
fine for both sudo articles

File: articles/cockpit-alp.asm.xml (hunk 1 above)
Comment (tomschr) [reply articles/cockpit-alp.asm.xml]: Thanks, it's in commit 6c5d3b9 now (together with a consistent ID in `<structure>`).

**Review by dariavladykina — APPROVED** [review]

**Review by lvicoun — APPROVED** [review]

**Review by tahliar — APPROVED** [review]
Looks good for network bond/team articles. Thank you!

**Review by dmpop — APPROVED** [review]
LGTM!

**Review by cwickert — APPROVED** [review]

**Pull request merged**